src/alda/parser/tokenize.clj

```diff
@@ -115,7 +114,7 @@
     (= :EOF x)
     parser
 
-    (#{\newline "\n"} x)
+    (#{\newline "\n"}x)
```

Comment:
Not sure if this was intentional, but could we back this change out, i.e. put the space back in?